### PR TITLE
#122 #9 Adding Premake support to OpenEAGGR 

### PR DESCRIPTION
**Pull request by OlekRaymond**
Added some helpful messages to a chosen system test to aid in debugging. Changed TestMacros.hpp to avoid underscores in google test.

### CODE REVIEW

File: EAGGRTestHarness/SystemTests/ImportExport/KmlExporterISEA4TSystemTest.cpp [hunk 1]
```diff
@@ -86,54 +127,43 @@ SYSTEM_TEST(KMLExporterISEA4T, IcosahedronAllFaces)
...
+      exporter.AddCell(*face16);
+      exporter.AddCell(*face17);
+      exporter.AddCell(*face18);
+      exporter.AddCell(*face19);
+  }
   // Export the KML
   std::ostringstream output;
   exporter.Export(output);
 
+  const std::string outputFilePath("OpenEAGGR/EAGGRTestHarness/TestData/ISEA4TIcosahedronAllFacesKmlExport.kml");
```
Comment (EmilySTyler) [inline EAGGRTestHarness/SystemTests/ImportExport/KmlExporterISEA4TSystemTest.cpp:157]: Ctrl-F for "ouput", should be "output"

File: EAGGRTestHarness/TestMacros.hpp [hunk 2]
```diff
@@ -33,12 +33,12 @@
...
+   TEST(UnitTest####TestBase, TestName)
 
 /// Used to group together all the system tests.
 #define SYSTEM_TEST(TestBase, TestName) \
-   TEST(SystemTest##_##TestBase, TestName)
+   TEST(SystemTest####TestBase, TestName)
 
 /// Used to group together all the evaluation tests.
 #define EVALUATION_TEST(TestBase, TestName) \
-   TEST(EvaluationTest##_##TestBase, TestName)
```
Comment (EmilySTyler) [inline EAGGRTestHarness/TestMacros.hpp:44]: Why did you change these? Just wondering

**Review by EmilySTyler — CHANGES_REQUESTED** [review]
Minor comments, also:

In EAGGR/premake:
- I think `removefiles { "**/3rdParty/**" }` should be `removefiles { "./3rdParty/**" }`
- The vpaths should have OpenEAGGR in front

File: EAGGRTestHarness/TestMacros.hpp (hunk 2 above)
Comment (OlekRaymond) [reply EAGGRTestHarness/TestMacros.hpp:44]: No underscored in google test names

**Comment by OlekRaymond** [comment]
> In EAGGR/premake:
> * I think `removefiles { "**/3rdParty/**" }` should be `removefiles { "./3rdParty/**" }`

Done
> * The vpaths should have OpenEAGGR in front

Nope? They are in the OpenEAGGR project, no need for OpenEAGGR/OpenEAGGR/HeaderFiles ?

File: EAGGRTestHarness/SystemTests/ImportExport/KmlExporterISEA4TSystemTest.cpp (hunk 1 above)
Comment (OlekRaymond) [reply EAGGRTestHarness/SystemTests/ImportExport/KmlExporterISEA4TSystemTest.cpp:157]: Done

**Review by EmilySTyler — APPROVED** [review]

**Pull request merged**